modules/nw-ovn-k-adminnetwork-policy-action-rules.adoc

```diff
@@ -0,0 +1,106 @@
+// Module included in the following assemblies:
+//
+// * networking/ovn-k-network-policy.adoc
+
+:_mod-docs-content-type: Concept
+[id="AdminNetworkPolicy-actions-for-rules"_{context}]
+= AdminNetworkPolicy actions for rules
+As an administrator, you can set `Allow`, `Deny`, or `Pass` as the action to your `AminNetworkPolicy` rules. Because OVN-Kuberenetes uses a tiered ACLs to evaluate network traffic rules, ANP rules allow you to set very strong network policy rules that can only be changed by an administrator modifying them, deleting the rule, or overriding them by setting a higher priority rule.
+
+[discrete]
+[id="AdminNetworkPolicy-allow-example"_{context}]
+== AdminNetworkPolicy Allow example
+The following `AdminNetworkPolicy` that is defined at priority 9 ensures all ingress traffic from the `monitoring` namespace towards any tenant (all other namespaces) in the cluster is allowed.
+
+
+.Example YAML file for a strong Allow rule
+[%collapsible]
+====
+[source,yaml]
+----
+apiVersion: policy.networking.k8s.io/v1alpha1
+kind: AdminNetworkPolicy
+metadata:
+  name: allow-monitoring
+spec:
+  priority: 9
+  subject:
+    namespaces: {}
+  ingress:
+  - name: "allow-ingress-from-monitoring"
+    action: "Allow"
+    from:
+    - namespaces:
+        namespaceSelector:
+          matchLabels:
+# ...
+----
+====
+The example is a strong `Allow` because it is non-overridable by all the parties involved. No tenants can block themselves from being monitored using `NetworkPolicies` and the monitoring tenant also has no say in what it can or cannot monitor.
```

Comment:
the allow rule is not technically accurate, and same for DENY as well, I won't comment this there, please change that
```suggestion
This is an example of a strong `Allow` ANP because it is non-overridable by all the parties involved. No tenants can block themselves from being monitored using `NetworkPolicies` and the monitoring tenant also has no say in what it can or cannot monitor.
```

Reply:
I think the the parts that are brought in from https://www.redhat.com/en/blog/using-adminnetworkpolicy-api-to-secure-openshift-cluster-networking have been changed over so the meaning changes a bit?

Reply:
Would `NetworPolicy` resources be ok as Alex asks about above? @tssurya

Reply:
I'd leave it as `NetworkPolicies` techncially speaking its fine, `NetworkPolicy` objects is better than resource if you really want to change them (hopefully they fit in all places)